backend/entityservice/tests/test_project.py

```diff
@@ -26,17 +26,16 @@ def _check_new_project_response_fields(new_project_data,
     assert len(new_project_data['update_tokens']) == expected_number_parties
 
 
-def test_simple_create_project(requests, number_parties_or_none):
+def test_simple_create_project(requests, valid_project_param):
```

Comment:
A bunch of these tests are also now parametrised to check output types other than `"mapping"`.